projects/demo/src/pages/recipes/placeholder/examples/2-phone/component.ts

```diff
@@ -48,12 +48,5 @@ export class PlaceholderDocExample2 {
         const initialValue = this.value || '+1 (';
 
         this.value = initialValue + PLACEHOLDER.slice(initialValue.length);
```

Comment:
Should we drop some code lines in this file ?

Reply:
No, they are necessary because it's custom focus/blur logic.